src/ServiceControl/Contracts/Operations/FailureDetails.cs

```diff
@@ -4,6 +4,11 @@ namespace ServiceControl.Contracts.Operations
 
     public class FailureDetails
     {
+        public FailureDetails()
+        {
+            TimeOfFailure = DateTime.UtcNow;
```

Comment:
There shouldn't, but just in case I decided to initialise it, because indexes depend on it